pytorch_lightning/utilities/distributed.py

```diff
@@ -0,0 +1,26 @@
+from functools import wraps
+import warnings
+
+
+def rank_zero_only(fn):
+
+    @wraps(fn)
+    def wrapped_fn(*args, **kwargs):
+        if rank_zero_only.rank == 0:
+            return fn(*args, **kwargs)
+
+    return wrapped_fn
+
+
+try:
+    # add the attribute to the function but don't overwrite in case Trainer has already set it
+    getattr(rank_zero_only, 'rank')
+except AttributeError:
+    rank_zero_only.rank = 0
```

Comment:
```suggestion
# add the attribute to the function but don't overwrite in case Trainer has already set it
rank_zero_only.rank = getattr(rank_zero_only, 'rank', 0)
```
I added the try catch block but this is probably more elegant :)